### ISSUE TITLE
Chromium doesn't accept some characters

### ISSUE THREAD

**Issue by emersion** [atx/wtype#31]
Works: "🐸", "!", ":", "a". Doesn't work: "é", "α", "#".

No idea why.

Workaround: https://github.com/Plagman/gamescope/pull/267/commits/d09516ba70dfa8c0c3ea39587b443965d616bdba

**Comment by atx** [comment]
Yep. Even worse - it looks like it _sometimes_ works and sometimes does not.
I can get it to work in the address bar by running `wtype α; wtype α`, which types a single `α`. Does not seem to work in HTML text fields at all. I am guessing there is some issue in how keymap reloads are handled.

I remember attempting to read through the chromium input handling code [last time it came up](https://github.com/atx/wtype/issues/25), but I just could not and gave up... Might either try later again or just implement the workaround.

**Comment by emersion** [comment]
Someone on Mastodon pointed me to this comment: https://gitlab.gnome.org/World/Phosh/squeekboard/-/issues/244#note_1237206

Note, the URL bar empties itself when switching the keymap it seems. Other text fields don't.

**Comment by emersion** [comment]
Okay, found a better workaround: https://github.com/Plagman/gamescope/commit/6764da38795a040e89bc66b544cf470fa52c5a4d

**Comment by cizra** [comment]
@emersion It's not really obvious how to apply that workaround - does it work? If it does, could you apply it to the repo (any downsides)?

**Comment by cjd** [comment]
I hit this problem as well and have a slightly different workaround.
Instead of typing directly I put the emoji in the clipboard and then use ctrl+V to paste that
ie:
echo -n "😎" | wl-copy; wtype -M ctrl V -m ctrl

That has worked for me in terminals and in chrome (both in address bar and in text areas)

**Comment by dotaxis** [comment]
Having an issue where wtype cannot type emojis into Discord or VSCode. This seems related? How can we apply the workaround?

**Comment by MoutonDemocrate** [comment]
Hi, following this. I would also appreciate to be able to type in Discord or VSCodium. How would the workaround be applicable there ?

**Comment by tillay** [comment]
I would like to know a workaround for Discord as well, please.

**Comment by thelamer** [comment]
Just ran into this on my stack, I'll get this patched, test some local builds and open a PR when I have it. 

Also thanks for maintaining this it saved my butt for international keyboard input in Wayland.
